style.css

```diff
@@ -18,6 +18,10 @@ a {
   text-decoration: none;
 }
 
+.desktop-nav {
```

Comment:
- [ ] **GOOD JOB 👏😀.** Everything is fixed as the last reviewer suggested. I suggest you improve your design. Please consider changing the colors of social icons in the footer on hovering 🍀

Screenshot
![image](https://user-images.githubusercontent.com/69011963/149539588-9b788fb3-a1fd-4a44-aa70-c139bfec2f2e.png)